BiweeklyMeetings/2022-08-16.md

```diff
@@ -0,0 +1,406 @@
+
+#### PR229
+[PR229 CIP-0045? | Decentralization: Using Pledge as a Bidding Param](https://github.com/cardano-foundation/CIPs/pull/229)
+
+**Matthias** -: As the fact that CIP 45 seems to incentivize pools to create, oh sorry, state pool operators to create many, many pools by giving them an optimal pledge to follow and is wrongly making the assumptions that one pool equals one individual. Whereas in practice, we know that to be true, there are samples that are run by a single individual, and therefore many of the assumptions of the rationale from that CIP crumbles a bit from that. So I haven't read yet through the comments that your author has made on that, but it seems that those points are being considered at least.
+
+**Matthias** -: There are also a few suggestions purely on the form itself, mainly that this, the CIP contains quite a lot of equations and visualizations, which are so far linked through an external Google drive as images. So I suggested the order to actually use the new latest support from GitHub that landed some months ago to actually display equations directly in GitHub, or at least to use an image, but hosted next to the CIP itself, that will simplify a bit. And the author has acknowledged those points and said it would be working on updating that according to the suggestions. So some more to come, I guess.
+
+**Robert Phair** -: As far as using the equations in CIPs and in GitHub content, the difficulty they ran into with another CIP 50, did they ever work out the presentation of those tech type equations on the mobile platform? Someone said they weren't working.
+
+**Matthias** -: So there is only Christoff on the chat commenting that. Yeah, they have done equations on GitHub, prone to some issues as you learn from CIP 50. So you are saying it's specifically on a mobile device, that it is not supported yet?
+
+**Robert Phair** -: That's what I think Christoff was saying.
+
+**Matthias** -: Which is probably okay. I mean, if you are reading a CIP of that magnitude on a mobile that's, I mean, you're pretty brave I would say.
+
+**Robert Phair** -: I get it. I hadn't thought of it that way. If they could go back to the original store. So if they want to see that kind of detail in the equations.
+
+**Matthias** -: Probably, it looks good in VS Code than Jupyter Notebooks, said Christoff. But then in GitHub and stuff, can't render, I guess, does not render on mobile in brackets, in desktop. I can't speak if, yeah, not, actually, since you are here, it'll faster than just writing on the chat. Can we just unmute you. Yes, it's done.
+
+**Christoff** -: Can you guys hear me?
+
+**Robert Phair** -: Yes. Hello?
+
+**Christoff** -: Oh, Hey everybody. Yeah. So as I learned in C50, yeah, the mobile definitely doesn't render, but then there's some brackets, especially curly brackets that doesn't render on GitHub even on desktop. So that's why we reverted back to basically screenshot equations from external source. Pictures.
```

Comment:
```suggestion
**Christoff** -: Oh, Hey everybody. Yeah. So as I learned in CIP 50, yeah, the mobile definitely doesn't render, but then there's some brackets, especially curly brackets that doesn't render on GitHub even on desktop. So that's why we reverted back to basically screenshot equations from external source. Pictures.
```